DAO.sol

```diff
@@ -696,6 +696,9 @@ contract DAO is DAOInterface, Token, TokenCreation {
         uint reward =
             (rewardToken[msg.sender] * DAOrewardAccount.accumulatedInput()) /
             totalRewardToken - DAOpaidOut[msg.sender];
+
+        if ( DAOrewardAccount.balance < reward) reward = DAOrewardAccount.balance;
```

Comment:
A note on our styling: The expression of the if branch should go to a new line.

But an even better suggestion here is to use the ternary operator:

`reward = DAOrewardAccount.balance < reward ? DAOrewardAccount.balance : reward;`